### PR TITLE
Rewrite Consumers

### PR DESCRIPTION
**Pull request by takeit**
## Fixes
SWP-1868, SWP-1869

## Proposed Changes

  - AnalyticsEventConsumer has been rewritten to use the Messenger abstraction
  - there are async queues created: `async_content_push`, `async_migration_content_push`, `async_analytics_event `, `async_analytics_export`, `async_webhooks`
  - added new supervisor config - no need to run the workers separately
  - WebhookConsumer was rewritten to use Messenger component
  - ContentPushConsumer was rewritten to use Messenger component
 - the logger service is automatically added to the send message middleware and logs are saved by default in `var/log/dev.log` in dev mode.
  - configured failed queue, failed messages will be stored in the db

**Usage:**
Run 
```bash
php bin/console messenger:consume async_content_push async_analytics_event async_webhooks async_analytics_export 
```

License: AGPLv3

### CODE REVIEW

File: src/SWP/Bundle/AnalyticsBundle/EventListener/AnalyticsEventListener.php [hunk 1]
```diff
@@ -16,92 +16,42 @@
...
-
-        if ($response->headers->has(self::TERMINATE_IMMEDIATELY)) {
-            $event->stopPropagation();
-        }
-    }
-
-    /**
-     * @param FinishRequestEvent $event
-     */
-    public function onKernelFinishRequest(FinishRequestEvent $event)
```
Comment (ahilles107) [inline src/SWP/Bundle/AnalyticsBundle/EventListener/AnalyticsEventListener.php]: why you removed those two events?

File: src/SWP/Bundle/CoreBundle/MessageHandler/AnalyticsEventHandler.php [hunk 2]
```diff
@@ -123,16 +93,9 @@ private function isHostMatchingTenant(string $host): bool
         return $host === $tenantHost;
     }
 
-    private function setTenant(Request $request): void
+    private function setTenant(string $httpReferrer): void
     {
-        $this->tenantContext->setTenant(
-            $this->tenantResolver->resolve(
-                $request->server->get('HTTP_REFERER',
-                    $request->query->get('host',
```
Comment (ahilles107) [inline src/SWP/Bundle/CoreBundle/MessageHandler/AnalyticsEventHandler.php]: this is used by 3d party events providers (like mobile app pageviews).

Comment (takeit) [reply src/SWP/Bundle/CoreBundle/MessageHandler/AnalyticsEventHandler.php]: it's there https://github.com/superdesk/web-publisher/pull/1041/files#diff-498b8ef8c7662ac5359ba7b5f6dcc861R44 in the new implementation

Comment (ahilles107) [reply src/SWP/Bundle/CoreBundle/MessageHandler/AnalyticsEventHandler.php]: 👍

**Pull request merged**